setup

```diff
@@ -27,14 +27,15 @@ verbose_output "Switched to $(pwd)"
 source oe-init-build-env
 verbose_output "Sourced the oe-init-build-env script"
 
-# Add the  meta-evm, meta-confidential-compute, meta-secure-core/meta-tpm2, meta-openembedded/meta-python, meta-openembedded/meta-oe layers meta-rust-bin meta-clang to bblayers.conf
+# Add the  meta-evm, meta-confidential-compute, meta-secure-core/meta-tpm2, meta-openembedded/meta-python, meta-openembedded/meta-oe layers meta-rust-bin meta-clang meta-observability to bblayers.conf
```

Comment:
comment seems a bit redundant

Reply:
I was thinking the same and wanted to remove it all together :thinking: